_styles/main.css

```diff
@@ -288,16 +288,70 @@ nav a:visited {
 }
 
 nav a:focus {
-    color: #08c;
-    fill: #08c;
     text-decoration: none;
 }
 
-nav a:hover,
-nav a:visited:hover {
+nav a:visited{
     opacity: 0.7;
 }
 
+/*color changes of .nav icon on focus*/ 
+
+nav .right li a[title="Facebook"]:focus{
+    color:#01579b; 
+    fill:#01579b;
+}
+
+nav .right li a[title="Google+"]:focus{
+    color:#d50000;
+    fill:#d50000;
+}
+
+nav .right li a[title="Twitter"]:focus{
+    color:#64b5f6;
+    fill:#64b5f6;
+}
+
+nav .right li a[title="Medium"]:focus{
+    color:#00AB82;
+    fill:#00AB82;
+}
+
+nav .right li a[title="Reddit"]:focus{
+    color:#FF4500;
+    fill:#FF4500;
+}
+
+nav .right li a[title="StackExchange"]:focus{
+    color:#2196f3;
+    fill:#2196f3;
+}
+
+/*color changes of .nav icon on hover*/
+
+nav .right li a[title="Facebook"]:hover{
```

Comment:
There is no need to specify all of these classes again. You should combine them with the above, so it reads like:
```
nav a[title="Facebook"]:focus,
nav a[title="Facebook"]:hover {
    color: #01579b;
    fill: #01579b;
}
```